### DISCUSSION TITLE
Kartendarstellung - Layer - Symbol für Metadaten

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#12] [Q&A] [answered]
Wie kann man das "?" -Symbol für den Metadatenlink durch ein "i" ersetzen?

**Comment by gis-eni** [comment, marked as answer]
Bilder und Icons liegen in der Regel in einem Unterverzeichnis von wwwroot (api oder Portal). Wohin ein Bild zeigt kann man über die Entwicklertools des Browsers herausfinden:

![image](https://user-images.githubusercontent.com/62870568/202664982-0751b20c-d261-4854-81e4-aeed73c61d2c.png)
(dieses Werkzeug wählen und danach über das entsprechende Icon fahren)

Wenn man Bilder austauscht, sollte man darauf achten, dass die Größe mit den Originalbild zusammen passt.
